src/applications/simple-forms/mock-simple-forms-patterns/components/ArrayBuilderSummaryCardList.jsx

```diff
@@ -0,0 +1,173 @@
+  removeDescription = () =>
+    'This will remove this item from your list of items.',
+  removeYesLabel = 'Yes, remove',
+}) => {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [currentIndex, setCurrentIndex] = useState(null);
+  const arrayData = get(formDataPath, formData);
+  const currentItem = arrayData?.[currentIndex];
+
+  if (!arrayData?.length) {
+    return null;
+  }
+
+  function showRemoveConfirmationModel(index) {
+    setCurrentIndex(index);
+    setIsModalVisible(true);
+  }
+
+  function hideRemoveConfirmationModel() {
+    setCurrentIndex(null);
+    setIsModalVisible(false);
+  }
+
+  function removeAction() {
+    const item = arrayData[currentIndex];
+    const arrayWithRemovedItem = arrayData.filter(
+      data => data.name !== item.name,
+    );
+    const newData = set(formDataPath, arrayWithRemovedItem, formData);
+
+    setFormData(newData);
+    hideRemoveConfirmationModel();
+  }
+
+  const Card = ({ item, index }) => (
+    <div className="vads-u-margin-bottom--2">
+      <va-card>
+        <CardContent item={item} />
+        <span className="vads-u-margin-top--2 vads-u-display--flex vads-u-justify-content--space-between">
+          <Link to={`${editBasePath}/${index}?=edit`}>
+            Edit
+            <i
+              aria-hidden="true"
+              className="fa fa-chevron-right vads-u-margin-left--1"
+            />
+          </Link>
+          {/* eslint-disable-next-line @department-of-veterans-affairs/prefer-button-component */}
+          <button
+            type="button"
+            className="va-button-link vads-u-color--secondary-dark"
+            onClick={() => showRemoveConfirmationModel(index)}
+          >
+            <i
+              aria-hidden="true"
+              className="fa fa-times vads-u-margin-right--1"
+            />
+            Remove
+          </button>
+        </span>
+      </va-card>
+    </div>
+  );
+
+  Card.propTypes = {
+    index: PropTypes.number.isRequired,
+    item: PropTypes.object.isRequired,
+  };
+
+  const Heading = `h${titleHeaderLevel}`;
+
+  return (
+    <div>
+      <Heading>{title}</Heading>
+      <div className="vads-u-display--flex vads-u-justify-content--space-between vads-u-flex-direction--column">
+        {arrayData?.length &&
+          arrayData.map((item, index) => (
+            <Card key={index} item={item} index={index} />
+          ))}
+      </div>
+      <VaModal
+        clickToClose
+        status="warning"
+        modalTitle={removeTitle}
+        primaryButtonText={removeYesLabel}
+        secondaryButtonText="No, cancel"
+        onCloseEvent={hideRemoveConfirmationModel}
+        onPrimaryButtonClick={removeAction}
+        onSecondaryButtonClick={hideRemoveConfirmationModel}
+        visible={isModalVisible}
+        uswds
+      >
+        {removeDescription(currentItem?.name)}
+      </VaModal>
+    </div>
+  );
+};
+
+const mapStateToProps = state => ({
```

Comment:
I think we need to map FormContext > onReviewPage here, so that the component's aware of when it's on the Review-page -- header-level needs to drop because the _Chapter_-title [accordion header] is also an H3 above it.

Reply:
Keep in mind that this Review-page dupe-H3 a11y-bug might be better handled within the Review-page instead.